services/mozilla-observatory/mozilla-observatory.tester.js

```diff
@@ -3,32 +3,27 @@
 const Joi = require('@hapi/joi')
 const t = (module.exports = require('../tester').createServiceTester())
 
-const validColors = ['brightgreen', 'green', 'yellow', 'orange', 'red']
 const isMessage = Joi.alternatives()
   .try(
     Joi.string().regex(/^[ABCDEF][+-]? \([0-9]{1,3}\/100\)$/),
     Joi.string().allow('pending')
   )
   .required()
 
-const isColor = Joi.string()
-  .valid(...validColors)
-  .required()
-
 t.create('request on observatory.mozilla.org')
+  .timeout(10000)
   .get('/grade-score/observatory.mozilla.org.json')
   .expectBadge({
     label: 'observatory',
     message: isMessage,
-    color: isColor,
```

Comment:
There's mocked tests below for every possible API response that include color validation, no need to duplicate that here with a catch-all color check